style.css

```diff
@@ -39,3 +39,40 @@
 .next.item.icon {
   font-size: 2em;
 }
+
+.calendar {
+  table-layout: fixed!important;
+}
+
+.calendar td {
+  display: table-cell!important;
+}
+
+.disabled {
+  background-color: rgba(177, 177, 177, 0.651)!important;
+}
+
+.disabled:hover {
```

Comment:
ha jól gondolom a cursor dologhoz nem feltétlenül kell a hover (hiszen úgy is csak akkor látszódik a kurzorból bármi ha hover)

másik: nem volt a bootstrapnek valami jó alkalmatos stílusa arra amit mi disablednek szerettünk volna?